Capri.Services/DiplomaCard/ParagraphGetter.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Capri.Web.ViewModels.Proposal;
+using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Wordprocessing;
+using Microsoft.Extensions.Localization;
```

Comment:
Missed Microsoft.Extensions.Localization.Abstractions dependency in csproj - PackageReference